src/hyp_map.rs

```diff
@@ -317,6 +255,21 @@ impl PrivateRegion {
     }
 }
 
+/// Generic Id names for each of the U-mode mapping slots.
+/// There is no mandated use for each of the slots, and caller can decide to map each of them
+/// readable or writable based on the requirement.
+#[derive(Copy, Clone)]
+pub enum UmodeSlotId {
+    A,
+    B,
+}
```

Comment:
should this go in `hyp_layout.rs`?

Reply:
Could go. Thought about it for about 30 seconds. SlotIDs and Slot Addrs are a specific of the implementation. But yeah, I'll move it, would make things cleaner.

Reply:
Moved.